<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DivinePreferences</class>
 <widget class="QWidget" name="DivinePreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="threadsLabel">
       <property name="text">
        <string>&amp;Worker threads:</string>
       </property>
       <property name="buddy">
        <cstring>threadsSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="threadsSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="memoryLabel">
       <property name="text">
        <string>Max &amp;memory (MB):</string>
       </property>
       <property name="buddy">
        <cstring>memorySpin</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="memorySpin">
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="pooledBox">
     <property name="text">
      <string>Disable &amp;pooled allocation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="reportBox">
     <property name="text">
      <string>Show &amp;report</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Program &amp;path:</string>
       </property>
       <property name="buddy">
        <cstring>pathEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pathEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="pathButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>threadsSpin</tabstop>
  <tabstop>memorySpin</tabstop>
  <tabstop>pooledBox</tabstop>
  <tabstop>reportBox</tabstop>
  <tabstop>pathEdit</tabstop>
  <tabstop>pathButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
